File: Day2/C/Ques8.c
#include<stdio.h>
int main(){
    int a,rev=0;
    printf("Enter a number:");
    scanf("%d",&a);
    int no=a;
    while (a>0){
        rev=(rev*10)+(a%10);
        a=a/10;
    }
    if(no==rev){
        printf("%d is a palindrome number!",no);
    }
    else if(no!=rev){
        printf("%d is not a palindrome number!",no);
    }
    return 0;
}

File: Day2/Python/Ques8.py
n=(int(input("Enter a number:")))
a=n
rev=0
while n>0:
    rev=rev*10+(n%10)
    n=n//10
print("Reverse of a number is:",rev)
if a==rev:
    print(a,"is a Palindrome number.")
elif a!=rev:
    print(a,"is a Palindrome number.")